iqdb_tagger/__main__.py

```diff
@@ -491,15 +491,16 @@ def get_hydrus_set(search_tags: List[str], client: Client) -> Iterator[Dict[str,
 @cli.command()
 @click.argument('tag', nargs=-1)
 @click.option('--access_key', help='Hydrus access key')
-def search_hydrus_and_send_url(tag: List[str], access_key: Optional[str] = None):
+@click.option('--hydrus_url', help='URL for hydrus client e.g. http://127.0.0.1:45869/')
+def search_hydrus_and_send_url(tag: List[str], access_key: Optional[str] = None, hydrus_url: Optional[str] = 'http://127.0.0.1:45869/'):
     """Search hydrus and send url."""
     # compatibility
     search_tags = tag
     if Client is None:
         print('Hydrus package is required')
         return
 
-    cl = Client(access_key)
+    cl = Client(access_key, hydrus_url)
```

Comment:
maybe set this as args here and other subcommand

```python
args = [access_key]
if hydrus_url:
    args.append(hydrus_url)
cl = Client(*args)
```
this way the old behaviour is kept even if hydrus module is changed